src/pages/MemoriesPage.js

```diff
@@ -0,0 +1,43 @@
+import styled from 'styled-components/macro'
+import MemoryItem from '../components/MemoryItem'
+import PropTypes from 'prop-types'
+
+MemoriesPage.propTypes = {
+  memories: PropTypes.arrayOf(
+    PropTypes.shape({ image: PropTypes.node, subtitle: PropTypes.string })
```

Comment:
I don't think `image` is `PropTypes.node`, instead, it is a string too (the path to the file).